alerts/google-gke/cpu-limit-utilization-containers-within-cluster.v1.json

```diff
@@ -0,0 +1,30 @@
+{
+  "displayName": "GKE Container - High CPU Limit Utilization (${CLUSTER_NAME} cluster)",
```

Comment:
@cocosheng , I really would like to use the template variable here, however, it makes the displayName in metadata.yaml and template different. Is that ok?

Reply:
discussed offline that this is ok since we're deprecating the field in metadata.yaml